src/common/kafka.ts

```diff
@@ -115,7 +115,7 @@ export async function consumeMessages(topic: string) {
         })
       );
       const cacheObject = JSON.parse(message.value?.toString() as string);
-      pdfCache.addToCollection(cacheObject?.id, {
+      pdfCache.addToCollection(cacheObject?.collectionId, {
```

Comment:
Should we add it to the cache if the `collectionId` does not exist? I would throw and error if the `cacheObject` does not exist or does not have the correct attributes.